scrunch/expressions.py

```diff
@@ -526,7 +527,8 @@ def category_ids(var_id, var_value, variables=variables):
         if len(subitems) == 2:
             if 'value' in subitems[1] and 'variable' in subitems[0]:
                 var_url = subitems[0]['variable']
-                if ds.variables.index[var_url]['type'] == 'multiple_response':
+                var_index = ds.variables.index
```

Comment:
This is more complicated than that. In an ideal scenario I would know if a variable exists in a Dataset and just GET is once. But the expression parser knows nothing about this. It just transforms names to possible variable urls. At this step of the process it could be a variable that doesn't exist in the Dataset.